hack/overrides/oidc.yml

```diff
@@ -0,0 +1,33 @@
+# oidc.yml - a docker-compose override that adds a OIDC auth to the stack
+#
+# There are 2 users and 1 group:
+# user1@example.com:user1pass;group1
+# user2@example.com:user2pass
+#
+# ref: https://hub.docker.com/r/qlik/simple-oidc-provider/
+# ref: https://docs.docker.com/compose/extends/
+#
+version: '3'
+
+services:
+  web:
+    environment:
+      # CONCOURSE_MAIN_TEAM_OIDC_USER: user1
+      CONCOURSE_MAIN_TEAM_OIDC_USER: John
+      # CONCOURSE_MAIN_TEAM_OIDC_GROUP: group1
+
+      CONCOURSE_OIDC_ISSUER: http://oidc:9000
```

Comment:
A little bump in the dev experience here but not much we can do about it :(
Can we add a comment to the top explaining that when logging in with this override file they'll be directed to `http://oidc:9000` and they'll need to replace `oidc` with `localhost` to continue logging in. I can see someone that doesn't used docker often being confused by the behaviour